acc_test/data/valid_multi_DC_provisioning_with_different_providers.tf

```diff
@@ -43,11 +43,11 @@ resource "instaclustr_cluster" "valid" {
       nodes_per_rack  = 1
     }
     provider = {
-      name = "AZURE"
+      name = "AZURE_AZ"
     }
     bundles {
       bundle = "APACHE_CASSANDRA"
-      version = "apache-cassandra-3.11.8.ic4"
+      version = "apache-cassandra-3.11.8.ic5"
```

Comment:
Can you use just the version for these ? e.g. "3.11.8"
This will reduce the need to always be updating these

Reply:
Tests failed when using just semver

Reply:
so fix them?